### ISSUE TITLE
Emoji implementation review

### ISSUE THREAD

**Issue by JustinDrake** [OpenBazaar/openbazaar-desktop#385]
It's really cool we have emojis, and I like the initial implementation. I'm comparing it to the Twitter implementation (see screenshot), which we should be able to reproduce with the Twitter library used.

![screen shot 2017-03-08 at 17 17 03](https://cloud.githubusercontent.com/assets/731743/23714993/1c2f78ae-0423-11e7-865f-6c17af9be106.png)

There are a few things missing:

1. Recently used emojis
2. Search
3. Left sidebar for categories
4. Bottom selector to change skin colour

I also noticed a few other stuff:

5. Some emojis are missing, e.g. flags
6. The order is not the same. For example the clocks in Twitters come right before the flags.

Are these differences with Twitter's implementation restrictions with https://github.com/twitter/twemoji?

**Comment by rmisio** [comment]
I agree a lot of those changes would be nice, but we timeboxed the development of this feature and what's in place is what we were able to get to. The rest of these are "really nice to have" enhancement that we'll likely look into after launch.

**Comment by hoffmabc** [comment]
We can break out these into feature issues and put them in the backlog for now.

**Comment by morebrownies** [comment]
@jjeffryes this could be another good issue to tag with "Help Wanted" as well. If Duo or someone in the community really wants to see this with the 2.0 release, I'm sure we'd be happy to take a PR for it.
